acl/types.proto

```diff
@@ -160,12 +160,12 @@ message EACLRecord {
   repeated Target targets = 4 [json_name="targets"];
 }
 
-// Extended ACL rules table. Defined a list of ACL rules additionally to Basic
-// ACL. Extended ACL rules can be attached to the container and can be updated
+// Extended ACL rules table. A list of ACL rules defined additionally to Basic
```

Comment:
Why? The meaning is that a list of extended ACls are defined in addition to basic ACL rules, isn't it? If we used `defines`, it would be like `a list defines` what? `additionally to basic rules`.